src/main/java/org/codehaus/mojo/taglist/FileAnalyser.java

```diff
@@ -122,7 +121,7 @@ public FileAnalyser(TagListReport report, List<TagClass> tagClasses) {
         log = report.getLog();
         sourceDirs = report.getSourceDirs();
         encoding = report.getInputEncoding();
-        locale = report.getLocale();
+        sourceFileLocale = report.getSourceFileLocale();
```

Comment:
From a interface point of view this class should not rely on the `TagListReport` class, but should have independent args/params.

Reply:
you are right ... but for me it is out of scope for this change

Reply:
maybe you should add a TODO here...